airbyte_cdk/sources/declarative/schema/dynamic_schema_loader.py

```diff
@@ -149,22 +182,26 @@ def get_json_schema(self) -> Mapping[str, Any]:
                 property_definition,
                 self.schema_type_identifier.type_pointer,
             )
+
+            value.update(
+                self.additional_property_fields_inferrer.infer(property_definition)
+                if self.additional_property_fields_inferrer
+                else {}
+            )
             properties[key] = value
 
-        transformed_properties = self._transform(properties, {})
+        transformed_properties = self._transform(properties)
 
         return {
             "$schema": "https://json-schema.org/draft-07/schema#",
             "type": "object",
-            "additionalProperties": True,
```

Comment:
💎 Nice. Appreciate that we're following JSON Schema standards for communicating this. 👍